src/components/Sweep/SweepExecute.tsx

```diff
@@ -0,0 +1,202 @@
+import { useCallback, useState, useEffect } from 'react';
+import { Form, Input, Button, notification, Typography } from 'antd';
+import { ethers } from 'ethers';
+import styled from 'styled-components';
+
+import { ethereumAddressValidator } from '../../utils';
+import { useMetamask } from '../../hooks/useMetamask';
+import { ChainId } from '../../constants';
+import { contractBalanceOf, contractTransferFrom, contractAllowance } from './useSweep';
+
+const { Text } = Typography;
+
+function SweepExecute() {
+  const [loading, setLoading] = useState(false);
+  const [valid, setValid] = useState(false);
+
+  const [fromAddress, setFromAddress] = useState('');
+  const [to, setTo] = useState('');
+  const [amount, setAmount] = useState(0);
+  const [balance, setBalance] = useState(0);
+  const [allowance, setAllowance] = useState(0);
+
+  const [form] = Form.useForm();
+
+  const { changeNetwork } = useMetamask();
+
+  const handleFromChange = useCallback(
+    (e) => {
+      setFromAddress(e.target.value);
+    },
+    [setFromAddress]
+  );
+
+  useEffect(() => {
+    const validAFromAddress = ethers.utils.isAddress(fromAddress);
```

Comment:
what is that `A` again?

Reply:
and also, ALWAYS when I point out an issue to fix, it's your responsibility to ask yourself "ok, do I need to fix that anywhere else?" and act accordingly. do not let me catch you fixing an issue in one place and ignoring it in another just because I didn't spell it out for you.
so in this case, the name of the constant is wrong